package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "thing-url-adapter",
```

Comment:
Don't change this. Future installs will be broken, i.e. settings won't transfer, etc.